test/integrations/destinations/reddit/processor/data.ts

```diff
@@ -126,7 +126,10 @@ export const data = [
                         screen_dimensions: {},
                       },
                       event_metadata: {
-                        item_count: 3,
+                        item_count: 2,
+                        currency: 'USD',
+                        value: 15,
+                        value_decimal: 0.15,
```

Comment:
what's expected out of `value_decimal` ? how 0.15 is interpreted, can we re-check docs ?

Reply:
Earlier this block was being omitted entirely  due to incorrect implementation, hence it is coming in tests

Reply:
From the docs shared by @yashasvibajpai , `value_decimal` is not just dividing by 100, it's for minor to major currency conversion